packages/cli-hermes/src/profileHermes/index.ts

```diff
@@ -0,0 +1,79 @@
+import {logger} from '@react-native-community/cli-tools';
+import {Config} from '@react-native-community/cli-types';
+import {downloadProfile} from './downloadProfile';
+
+type Options = {
+  verbose: boolean;
+  fileName?: string;
+  raw?: boolean;
+  sourcemapPath?: string;
+  generateSourcemap?: boolean;
+};
+
+async function profileHermes(
+  [dstPath]: Array<string>,
+  ctx: Config,
+  options: Options,
+) {
+  try {
+    logger.info(
+      'Downloading a Hermes Sampling Profiler from your Android device...\n',
+    );
+    if (!options.fileName) {
+      logger.info('No filename is provided, pulling latest file\n');
+    }
+    if (options.verbose) {
+      logger.setVerbose(true);
+    }
+    await downloadProfile(
+      ctx,
+      dstPath,
+      options.fileName,
+      options.sourcemapPath,
+      options.raw,
+      options.generateSourcemap,
+    );
+  } catch (err) {
+    logger.error(`Unable to download the Hermes Sampling Profile.\n${err}`);
```

Comment:
we tend to `throw CLIError(message, err)` in such cases

Reply:
Yeah. I already left an example few comments earlier.